modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/ConfigurationAsmGenerator.java

```diff
@@ -2611,7 +2622,13 @@ private BytecodeBlock treatSourceForConstruct(
         
         return codeBlock;
     }
-    
+
+    private String polyConfTypeIsNotDefinedMessage(Field polymorphicIdField) {
```

Comment:
Empty description, let's rename to `polymorphicTypeNotDefinedErrorMessage`

Reply:
Renamed the method.

The method name is now descriptive by itself. What kind of a description would you like to have in addition?